presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java

```diff
@@ -100,11 +103,26 @@ public void testExistingView()
     public void testShowCreateView()
     {
         onHive().executeQuery("DROP VIEW IF EXISTS hive_show_view");
-
         onHive().executeQuery("CREATE VIEW hive_show_view AS SELECT * FROM nation");
 
-        // view SQL depends on Hive distribution
-        assertThat(query("SHOW CREATE VIEW hive_show_view")).hasRowsCount(1);
+        String showCreateViewSql = "SHOW CREATE VIEW %s.default.hive_show_view";
+        String expectedResult = "CREATE VIEW %s.default.hive_show_view AS\n" +
+                "SELECT\n" +
+                "  \"n_nationkey\"\n" +
+                ", \"n_name\"\n" +
+                ", \"n_regionkey\"\n" +
+                ", \"n_comment\"\n" +
+                "FROM\n" +
+                "  \"default\".\"nation\"";
+
+        QueryResult actualResult = query(format(showCreateViewSql, "hive"));
+        assertThat(actualResult).hasRowsCount(1);
+        assertEquals((String) actualResult.row(0).get(0), format(expectedResult, "hive"));
+
+        // Validate the translated view sql for a catalog name other than "hive"
+        actualResult = query(format(showCreateViewSql, "hive_with_external_writes"));
```

Comment:
Might be worth calling out in the comment that the `hive_with_external_writes` points to the same HMS instance